statsmodels/tsa/exponential_smoothing/base.py

```diff
@@ -495,6 +491,321 @@ def cov_params_approx(self):
             self._cov_approx_complex_step, self._cov_approx_centered
         )
 
+    def test_serial_correlation(self, method, lags=None):
```

Comment:
If the docstrings are identical you can use `@Appender` to append the existing docstring. Helps if/when there are typos.

Reply:
I plan to remove this when working on #6881.